qemu/tests/cfg/live_snapshot.cfg

```diff
@@ -60,8 +60,6 @@
             initrd = initrd.img
             force_create_image_stg = yes
             image_aio = threads
-            unattended_delivery_method = cdrom
-            cdroms += " unattended"
```

Comment:
IMO, it's not a good idea to delete it directly.

Reply:
Yes, this change is not correct, as i miss Linux guest.
For  windows guest,  add "unattended" to "cdroms" only is not enough, "cdrom_unattended" need be defined for different guest too, such as "images/win10-64/autounattend.iso"
and "unattended_delivery_method = cdrom" will replaced by "floppy" that defined in Windows.cfg in staf.
For windows guest, no matter delete it or not,  this case use answer.vfd, not unattended.iso.
So, it it needless, can be removed.

For Linux guest, it can't be removed, "cdrom_unattended" are defined in guest-os files, it is need for test.
 
This is a problem for case "with_installation",  not must for this patch.
i will keep it in this patch,  and we can send new patch to fix the problem above. thanks a lot!

Reply:
Agree.